conftest.py

```diff
@@ -5,7 +5,7 @@
 from fastapi.testclient import TestClient
 from pytest import Parser, fixture
 
-from stapi_fastapi.api import StapiRouter
+from stapi_fastapi.main_router import MainRouter
```

Comment:
Naming things is the hardest. I don't know that I have any great names for this, but I do wonder if we want something more descriptive so when implementations import this class it's clear what it is?